frameit/lib/frameit/config_parser.rb

```diff
@@ -64,32 +64,27 @@ def validate_values(values)
         if value.kind_of?(Hash)
           validate_values(value) # recursive call
         else
-          if key == 'font'
+          case key
+          when 'font'
             UI.user_error!("Could not find font at path '#{File.expand_path(value)}'") unless File.exist?(value)
-          end
-
-          if key == 'fonts'
+          when 'fonts'
             UI.user_error!("`fonts` must be an array") unless value.kind_of?(Array)
 
             value.each do |current|
               UI.user_error!("You must specify a font path") if current.fetch('font', '').length == 0
               UI.user_error!("Could not find font at path '#{File.expand_path(current.fetch('font'))}'") unless File.exist?(current.fetch('font'))
               UI.user_error!("`supported` must be an array") unless current.fetch('supported', []).kind_of? Array
             end
-          end
-
-          if key == 'background'
+          when 'background'
             UI.user_error!("Could not find background image at path '#{File.expand_path(value)}'") unless File.exist? value
-          end
-
-          if key == 'color'
+          when 'color'
             UI.user_error!("Invalid color '#{value}'. Must be valid Hex #123123") unless value.include?("#")
-          end
-
-          if key == 'padding'
-            unless value.kind_of?(Integer) || value.split('x').length == 2
-              UI.user_error!("padding must be type integer or pair of integers of format 'AxB'")
+          when 'padding'
+            unless value.kind_of?(Integer) || value.split('x').length == 2 || (value.end_with?('%') && value.to_f > 0)
+              UI.user_error! "padding must be type integer or pair of integers of format 'AxB' or a percentage"
```

Comment:
Perhaps this error message could indicate that this is a percentage of screen size? Also, not necessarily required right now, but is there any value for a pair of percentages to specify separate width/height multipliers?